lore/io/__init__.py

```diff
@@ -35,7 +36,11 @@
         if options.get('url') == '$DATABASE_URL':
             logger.error('$DATABASE_URL is not set, but is used in config/database.cfg. Skipping connection.')
         else:
-            vars()[section.lower()] = Connection(name=section.lower(), **options)
+            if 'urls' in options:
+                vars()[section.lower()] = MultiConnectionProxy(name=section.lower(), **options)
```

Comment:
May be valuable to specify which connection to stick to. This could be useful In the case of an incident when we are asked to stick to a certain replica. What do you think?

Reply:
Can we just remove a bad one from the environment variable (e.g. $SHOPPERS_DATABASE_URLS) and relaunch the service? The `urls` can take single URL as well. Will that work?

In Ruby land, we use Makara, which can automatically flag a bad connection and blacklist it for a certain period of time. But I think this capability is out of the scope of this PR, but certainly doable in the future.

Reply:
Think that should work. Thanks.